src/styles/globals.css

```diff
@@ -16,3 +16,12 @@
 	background-image: linear-gradient(to right, #1c7ed6, #4dabf7, #339af0, #1c7ed6);
 	animation: gradient-text-animation 4s infinite linear;
 }
+
+.subitem {
```

Comment:
Not sure if we should make changes to global css unless absolutely necessary.

@Lxkas  please confirm this one

Reply:
Yeah, don't touch the global css file if not absolutely necessary pls